experimental/dds/tree/src/EditLog.ts

```diff
@@ -269,8 +269,11 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 			this.registerEditAddedHandler(handler);
 		}
 
-		for (const handler of editEvictionHandlers) {
-			this.registerEditEvictionHandler(handler);
+		if (targetLength !== Infinity) {
```

Comment:
While you're at it, should we assert that `targetLength` and `evictionFrequency` are not negative (and, not 0, I think?). It might be that things work out okay with negative numbers given the math below, but I'd rather just fail and not have to think about it :)

Reply:
If we do allow those values to be negative without failing, we should have tests for those negative values too ;P